tools/rapids-configure-conda-channels

```diff
@@ -1,11 +1,29 @@
 #!/bin/bash
 # A utility script that configures conda channels
 
+conda_channel_in_config() {
+  channel_id=${1:?err}
+  conda config --json --get channels \
+  | jq -r --arg c "${channel_id}" '."get"."channels" | any(. == $c )'
+}
+
+# Only try to run 'conda config --remove' if the channel exists in the config.
+# This is here to avoid errors if this script is invoked multiple times in the same
+remove_conda_channel() {
+  channel_id=${1:?err}
+  in_config=$(conda_channel_in_config "${channel_id}")
+  if [[ "${in_config}" == "true" ]]; then
+    conda config --system --remove channels "${channel_id}"
+  else
+    echo "[rapids-configure-conda-channels] channel '${channel_id}' not found via 'conda config --get channels'"
```

Comment:
Proposing adding this so we have a chance of catching the "there was a typo in the channel name" case.

In theory this should never show up in CI logs unless there's a problem, since this script should only be run once during CI jobs.